weboffice/collabora.yml

```diff
@@ -14,6 +14,7 @@ services:
 
   collaboration:
     image: ${OC_DOCKER_IMAGE:-opencloudeu/opencloud-rolling}:${OC_DOCKER_TAG:-latest}
+    user: ${OC_CONTAINER_UID_GID:-1000:1000}
```

Comment:
I do not know how collabora reacts with that change. I think we can omit that, because collabora is not accessing opencloud volumes.